include/btllib/nthash.hpp

```diff
@@ -17,6 +17,12 @@
 
 namespace btllib {
 
+const std::string NTHASH_VERSION = "ntHash2";
```

Comment:
Since you added this variable, the `HASH_FN` variable in `bloom_filter.hpp` and `counting_bloom_filter.hpp` have been made redundant. `HASH_FN` declaration should be removed and wherever it's used, we should substitute `NTHASH_VERSION`.

Reply:
We can just do `HASH_FN = NTHASH_VERSION` wherever needed. Changing `NTHASH_VERSION` to something more intuitive like `NTHASH_FN_NAME` may be a good idea here. I'll do the substitutions.

Reply:
Sounds good, I like the suggestion.

Reply:
Renamed in https://github.com/bcgsc/btllib/commit/0b6952a3218ea6de566f3d96793db5e4a0b512ca and substituted in https://github.com/bcgsc/btllib/commit/b3b46ea296014279e80de836a98f6736bb58d543. I only found one line that needed substitution. Please let me know if there's more.